nebula/ui/components/MZCheckBoxRow.qml

```diff
@@ -60,10 +60,22 @@ RowLayout {
         MZTextBlock {
             id: subLabel
 
+            font.pixelSize: label.visible ? MZTheme.theme.fontSizeSmall : MZTheme.theme.fontSize
             Layout.fillWidth: true
             text: subLabelText
             visible: !!subLabelText.length
             wrapMode: Text.WordWrap
+            verticalAlignment: label.visible ? Text.AlignTop : Text.AlignVCenter
+            Layout.topMargin: label.visible || (!label.visible && lineCount > 1) ? 0 : 2 //accounts for the topMargin of the MZCheckbox
+
+            MZMouseArea {
+                enabled: !label.visible
+                anchors.fill: undefined
+                width: parent.implicitWidth
+                height: parent.implicitHeight
+                propagateClickToParent: false
+                onClicked: checkBox.clicked()
+            }
```

Comment:
Previously, all of our checkboxes had a label and a sublabel, but the checkboxes in the onboarding mocks only have one. Since the label of these checkboxes in onboarding look more akin to the sublabel of our regular checkboxes, I opted to hide the regular label and just show the sublabel, and added a mouse area so clicking on the sublabel (like clicking on the label of all other checkboxes) checks/unchecks the box. Normally our sublabels aren't clickable. This was discussed with @peterbenvenuto 

Alternatively, we could remove this extra mouse area and modify properties of the label to get the look of the sublabel (font size, color, line height, etc)

Reply:
Thanks for the explanation. Since label already has the capability for visiblibility to be turned off (visible: !!labelText.length), this looks fine I think.